packages/@react-spectrum/picker/src/Picker.tsx

```diff
@@ -46,6 +47,7 @@ import {useProvider, useProviderProps} from '@react-spectrum/provider';
 import {useSelectState} from '@react-stately/select';
 
 function Picker<T extends object>(props: SpectrumPickerProps<T>, ref: DOMRef<HTMLDivElement>) {
+  props = useSlotProps(props, 'picker');
```

Comment:
we may need to do a similar thing here so you can get a class on the right elements